tests/hpc/spack_slave.pm

```diff
@@ -25,9 +25,13 @@ sub run ($self) {
     type_string("$testapi::password\n");
     barrier_wait('CLUSTER_PROVISIONED');
     barrier_wait('MPI_SETUP_READY');
-    assert_script_run "spack load $mpi";
-    script_run "module av";
-
+    if (check_var('HPC_LIB', 'boost')) {
+        assert_script_run 'source /usr/share/spack/setup-env.sh';
```

Comment:
Also here.

Reply:
As per last reply too. lets keep it as it is if there is no problem